projects/storefrontapp/src/app/spartacus/features/quote-feature.module.ts

```diff
@@ -54,11 +54,6 @@ import {
         },
       },
     }),
-    provideConfig(<FeaturesConfig>{
-      features: {
-        storeFrontLibCardParagraphTruncated: true,
```

Comment:
the removed flag has a value `true` while the default value of `storeFrontLibCardParagraphTruncated` inside `default-feature-toggles.ts` is `true`. Is this intentional?

Reply:
Oups. My bad. Good catch.